vector/build.gradle

```diff
@@ -24,12 +24,16 @@ static def getGitTimestamp() {
 }
 
 static def generateVersionCodeFromTimestamp() {
-    // It's unix timestamp divided by 10: It's incremented by one every 10 seconds.
-    return (getGitTimestamp() / 10).toInteger()
+    // It's unix timestamp, minus timestamp of October 3rd 2018 (first commit date) divided by 100: It's incremented by one every 100 seconds.
+    // plus 20_000_000 for compatibility reason with the previous timestamp
+    // Note that the result will be multiplied by 10 when adding the digit for the arch
+    return ((getGitTimestamp() - 1_538_524_800 ) / 100).toInteger() + 20_000_000
 }
 
 def generateVersionCodeFromVersionName() {
-    return versionMajor * 1_00_00 + versionMinor * 1_00 + versionPatch
+    // plus 4_000_000 for compatibility reason with the previous timestamp
```

Comment:
There's no previous timestamp in this case.

Reply:
I'm thinking about @krombel repository

Reply:
The logic is correct, I think, just the comment should not reference a timestamp for the release build but just compatibility with the previous version code generation? Or am I missing something here?

Reply:
yes, you're right, I update the comment